src/vs/editor/browser/widget/diffReview.ts

```diff
@@ -178,6 +178,10 @@ export class DiffReview extends Disposable {
 				|| e.equals(KeyCode.Space)
 				|| e.equals(KeyCode.Enter)
 			) {
+				if (this._diffEditor.getContainerDomNode().classList.contains('screen-reader-interactive-editor-widget')) {
+					// We don't want to show the underlying editors when the diff editor is in inline mode
```

Comment:
This is a conceptual layer breaker and some real API is needed instead

Reply:
Ok, will add